modelscan/models/h5/scan.py

```diff
@@ -54,15 +54,17 @@ def _scan_keras_h5_file(
     def _get_keras_h5_operator_names(source: Union[str, Path]) -> List[str]:
         # Todo: source isn't guaranteed to be a file
         with h5py.File(source, "r") as model_hdf5:
-            lambda_code = [
-                layer.get("config", {}).get("function", {})
-                for layer in json.loads(model_hdf5.attrs["model_config"])["config"][
-                    "layers"
-                ]
-                if layer["class_name"] == "Lambda"
-            ]
+            model_config = json.loads(model_hdf5.attrs["model_config"])
```

Comment:
It might be a good idea here to handle JSONDecoderError in case parsing JSON fails for `model_config`.

Reply:
Yes, thank you, I have added `try-except` block to handle potential errors during JSON loading.